src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -21,7 +21,8 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <ContainerName>$(TestProduct)-$(Branch)-$(BuildMoniker)</ContainerName>
+    <TargetOS>$(TargetQueue.Replace('.', '-'))</TargetOS>
```

Comment:
This depends on the target queue having the same name as the target OS, which while true today might not be in the future.  It also adds drags in the full name of the queue which isn't really relevant (e.g. Windows.PerfInfraTest).  If we don't want to impose people putting the OS name in the BuildMoniker property I would rather add a dedicated TargetOS property to be passed explicitly.